hack/update-codegen.sh

```diff
@@ -89,12 +89,15 @@ function generate_group() {
   echo "generating conversions"
   bazel run @io_k8s_code_generator//cmd/conversion-gen -- \
     --input-dirs ${APIS_PKG}/${GROUP_NAME},${APIS_PKG}/${GROUP_NAME}/${VERSION} \
+    --extra-peer-dirs "k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/conversion,k8s.io/apimachinery/pkg/runtime" \
     --output-base ${GEN_TMPDIR} \
     --output-file-base zz_generated.conversion
 
   rsync -a "${GEN_TMPDIR}/k8s.io/cluster-registry/" "${SCRIPT_BASE}/${REPO_DIRNAME}"
 }
 
+bazel fetch //:genfiles_dummy
```

Comment:
This is already performed as one of the first things in the `gopath_from_workspace.sh` script called below.

Reply:
That's true. I suppose that this script doesn't actually do anything with the fetched dependencies, so I haven't actually made the two scripts loosely coupled.